docs/usage/self-hosted-configuration.md

```diff
@@ -233,6 +233,18 @@ This feature is useful for users who want Renovate to only work on repositories
 The `autodiscoverProjects` config option takes an array of minimatch-compatible globs or RE2-compatible regex strings.
 For more details on this syntax see Renovate's [string pattern matching documentation](./string-pattern-matching.md).
 
+## autodiscoverRepoOrder
+
+The order method for autodiscover server side repository search.
```

Comment:
default values should be documented

Reply:
They should be present in the config options table now.